files/ru/web/media/formats/webrtc_codecs/index.md

```diff
@@ -65,7 +66,7 @@ VP8, который мы [описывали в общем](/ru/docs/Web/Media/F
 
 Поддержка профиля AVC Constrained Baseline (`CB`) требуется во всех полностью совместимых реализациях WebRTC. `CB` является подмножеством основного профиля и специально разработан для приложений с низкой сложностью и малой задержкой, таких как мобильное видео и видеоконференции, а также для платформ с более низкими возможностями обработки видео..
 
-Наш [обзор AVC](</ru/docs/Web/Media/Formats/Video_codecs#AVC_(H.264)>) и его функциональности найдёте в основном руководстве по видеокодекам.
+Наш [обзор AVC](/ru/docs/Web/Media/Formats/Video_codecs#AVC_(H.264)) и его функциональности найдёте в основном руководстве по видеокодекам.
```

Comment:
We actually do need to leave the angle brackets in!  Because the URL itself contains parenthesis, we need to escape the link:

```suggestion
Наш [обзор AVC](</ru/docs/Web/Media/Formats/Video_codecs#AVC_(H.264)>) и его функциональности найдёте в основном руководстве по видеокодекам.
```